node/sc/api_bridge.go

```diff
@@ -253,6 +254,21 @@ func (sb *SubBridgeAPI) GetAnchoring() bool {
 	return sb.subBridge.GetAnchoringTx()
 }
 
+func checkBridgeContractVersion(cBridge, pBridge *bridge.Bridge) bool {
+	errStr := "Bridge contract version mistmatched"
```

Comment:
Can we reuse the global variable `ErrBridgeContractVersionMismatch` here instead of redefining the same string?

Reply:
Refactored. PTAL.